app/(public)/revision/[...revisions]/page.tsx

```
@@ -53,23 +53,27 @@ export default async function Page({
+    banner?: string;
   }>;
 }) {
+  let type: "draft" | "live" | "revision" = "revision";
+  let id = "";
+  let lang = "";
+  let label = "";
   const { isEnabled } = await draftMode();
   const nextParams = await params;
   const search = await searchParams;
-
+  const banner = search.banner === "false" ? false : true;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)